src/I_Ecobee1.xml

```diff
@@ -954,7 +954,7 @@ decompress_lzo_file() {
       if isEmsThermostat(lul_device) then
         return away and setOccupied(session, selection, lul_device, false) or resumeProgram(session, selection, 3)
       else
-        return setClimateHold(session, selection, lul_device, away and "away" or "home")
+        return setClimateHold(session, selection, lul_device, away and "away" or "home" or "sleep" or "vacation")
```

Comment:
I don't think Lua operators work this way.  The `and` and `or` in this context are like `if` and `else` in other languages.